### PR TITLE
Made changes on the layouts and the dashboard

### PR DESCRIPTION
**Pull request by Dhanielgeek**

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  name: string;
+  icon: React.ReactElement;
+  path: string;
+}
+
+interface SidebarProps {
+  active: boolean;
+  setActive: (active: boolean) => void;
+}
+
```
Comment (codewithbayo) [inline src/components/Sidebar.tsx:21]: can you move this to the interface folder

File: src/Styles/custom.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      }
+    }
+  }
+}
+.menu_bar {
+  @include flex_center();
+  @media screen and (min-width: 768px) {
+    display: none;
+  }
+}
```
Comment (codewithbayo) [inline src/Styles/custom.scss:116]: why are you using external css is it because of ant design

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      path: "account/security",
+    },
+    {
+      label: "Notifications",
+      icon: <FaRegBell className="text-2xl" />,
+      path: "/user/overview",
+      mobileOnly: true,
+    },
+  ];
+
```
Comment (codewithbayo) [inline src/components/Header.tsx:79]: you can also move this to another file incase we want to add more content on the sidebar

File: src/components/Welcome.tsx [hunk 4]
```diff
@@ -41,7 +41,7 @@ const Welcome = () => {
           transition={{ delay: 0.8, duration: 0.5 }}
           className="w-full h-[35%] flex justify-around items-center flex-col"
         >
-          <p className="text-2xl font-semibold">Welcome to MarketPlace</p>
+          <p className="text-2xl font-semibold">Welcome to maarketPlaace</p>
```
Comment (codewithbayo) [inline src/components/Welcome.tsx:44]: <p className="text-2xl font-semibold">Welcome to maarketplaace</p>
you can make that p small on maarketplaace to be small later

File: src/components/Sidebar.tsx (hunk 1 above)
Comment (Dhanielgeek) [reply src/components/Sidebar.tsx:21]: Okay  sure

File: src/Styles/custom.scss (hunk 2 above)
Comment (Dhanielgeek) [reply src/Styles/custom.scss:116]: yes

File: src/components/Header.tsx (hunk 3 above)
Comment (Dhanielgeek) [reply src/components/Header.tsx:79]: okay will do

**Review by youngmentor — APPROVED** [review]

**Pull request merged**